lldb/source/Plugins/Language/Swift/SwiftUnsafeTypes.cpp

```diff
@@ -149,7 +232,7 @@ class UnsafeBufferPointerSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
   uint8_t m_ptr_size;
   lldb::ByteOrder m_order;
 
-  SwiftUnsafeBufferPointer m_unsafe_ptr;
+  SwiftUnsafeBufferPointer *m_unsafe_ptr;
```

Comment:
You are leaking this SwiftUnsafeBufferPointer now.  Probably better to make this a unique_ptr<SwiftUnsafeBufferPointer>, then it will get cleaned up automatically.